src/__tests__/clowder.test.ts

```diff
@@ -125,6 +125,36 @@ describe('Clowder Class', () => {
     expect(nextItem).toBeUndefined();
   });
 
+  it('should return an item from missing if catToSelect is "unvalidated", no unvalidated items remain, and returnUndefinedOnExhaustion is false', () => {
+    const clowderInput: ClowderInput = {
+      cats: {
+        cat1: { method: 'MLE', theta: 0.5 },
+      },
+      corpus: [
+        createMultiZetaStimulus('0', [createZetaCatMap(['cat1'])]), // Validated item
+        createMultiZetaStimulus('1', [createZetaCatMap([])]), // Unvalidated item
```

Comment:
Since we're testing a random draw from `missing`, can we include multiple options and then test a few times.

```suggestion
        createMultiZetaStimulus('0', [createZetaCatMap(['cat1'])]), // Validated item
        createMultiZetaStimulus('1', [createZetaCatMap([])]), // Unvalidated item
        createMultiZetaStimulus('2', [createZetaCatMap([])]), // Unvalidated item
```